Package.swift

```diff
@@ -37,8 +38,9 @@ let package = Package(
     ] + swiftLintPackage(),
     targets: [
         .target(
-            name: "SpeziOnboarding",
+            name: "SpeziConsent",
             dependencies: [
+                .target(name: "SpeziOnboarding"),
```

Comment:
Is there any way that we can implement SpeziConsent without depending on SpeziOnboarding? What are the current types that are used on Consent? If so, could they move into a shared infrastructure target for both targets within this package?

If we can achieve this, we could even make this a non-breaking change by just `@_exported` import SpeziConsent within the SpeziOnboarding module for now until we tag a new major version. What do you think about that?

Reply:
The `OnboardingConsentView` has a dependency on `OnboardingView` and `OnboardingTitleView`. Additionally, in `ConsentDocument+Export.swift` there is a dependency on `OnboardingTitleView`.

Since both of those are very simple SwiftUI views, this may be feasible, yes!

Reply:
Sounds great; let's try to extract that in a shared small target so we can just integrate consent in the onboarding module as an `@_exported` import; will make it easy to separate out the code now but will allow us to easily remove that and tag a major version where this is moved to a separate repo in the future 🚀

Reply:
done